metaflow/sidecar/sidecar.py

```diff
@@ -0,0 +1,42 @@
+from tracemalloc import is_tracing
+from .sidecar_subprocess import SidecarSubProcess
+from .sidecar_messages import Message, MessageTypes
+
+
+class Sidecar(object):
+    TYPE = ""
+
+    def __init__(self, flow, env):
+        self.sidecar_process = None
+
+    def start(self):
+        if not self._is_active and self.get_sidecar_worker_class() is not None:
```

Comment:
When is `self.get_sidecar_worker_class()` `None`?

Reply:
for the Null* ones. This is an optimization of sorts -- we don't start the subprocess. It was like that before so kept that behavior.